extensions/wikia/AdEngine/js/utils/env.js

```diff
@@ -0,0 +1,13 @@
+define('ext.wikia.adEngine.utils.env', [
+	'wikia.window'
+], function (win) {
+	'use strict';
+
+	function isDevEnvironment() {
+		return /.+wikia-dev\.com$/.test(win.location.host);
```

Comment:
I second @nandy-andy opinion here.

I'd create a custom ResourceLoader module with PHP logic that will emit config for the current environment only based on `$wgWikiaEnvironment` value.

Please remember that we currently have more than just prod and dev environments:

```php
define( 'WIKIA_ENV_DEV', 'dev' );
define( 'WIKIA_ENV_PREVIEW', 'preview' );
define( 'WIKIA_ENV_PROD', 'prod' );
define( 'WIKIA_ENV_SANDBOX', 'sandbox' );
define( 'WIKIA_ENV_VERIFY', 'verify' );
define( 'WIKIA_ENV_STABLE', 'stable' );
define( 'WIKIA_ENV_STAGING', 'staging' );
```